core/services/gateway/handlers/vault/aggregator.go

```diff
@@ -75,13 +76,13 @@ func (a *baseAggregator) validateUsingQuorum(don capabilities.DON, resps map[str
 			maxShaToCount = shaToCount[sha]
 		}
 		if shaToCount[sha] >= requiredQuorum {
-			return r, nil
+			return &r, nil
 		}
 	}
 
 	remainingResponses := len(don.Members) - len(resps)
 	if maxShaToCount+remainingResponses < requiredQuorum {
-		return nil, errQuorumUnobtainable
+		return nil, errors.New(errQuorumUnobtainable.Error() + ". RequiredQuorum=" + strconv.Itoa(requiredQuorum) + ". maxShaToCount=" + strconv.Itoa(maxShaToCount) + " remainingResponses=" + strconv.Itoa(remainingResponses))
```

Comment:
Hmmmm maybe we should wrap the error here? I wonder if we ever `errors.Is(errQuorumUnobtainable, ...)`? In that case this has broken that

Reply:
(Doesn't look like we do)